### DISCUSSION TITLE
Using the RCareWorld environment using SSH

### DISCUSSION THREAD

**Discussion by rajas1310** [empriselab/RCareWorld#104] [Help - Setting up / Configuring] [answered]
I have successfully installed the RCareWorld on a remote machine. I want to work remotely hence will be using SSH for connecting to the remote host. Is there a way I can view the Unity simulation window/tab on my local machine? 

I tried editing the following lines  `/etc/ssh/sshd_config` :
```
X11Forwarding yes
X11DisplayOffset 10
```
Followed by `systemctl restart sshd`.
But this did not work.

**Comment by YoruCathy** [comment, marked as answer]
check this post https://discussions.unity.com/t/run-mlagents_envs-unityenvironment-from-remote-ssh-login/816539
